pySDC/projects/DAE/sweepers/fully_implicit_DAE.py

```diff
@@ -31,6 +31,12 @@ def __init__(self, params):
         # call parent's initialization routine
         super(fully_implicit_DAE, self).__init__(params)
 
+        msg = f"Quadrature type {self.params.quad_type} is not implemented yet. Use 'RADAU-RIGHT' instead!"
+        if not self.coll.right_is_node:
+            raise ParameterError(msg)
+        elif self.coll.right_is_node and self.coll.left_is_node:
+            raise ParameterError(msg)
```

Comment:
In case of `'LOBATTO'` and `'RADAU-LEFT'` we have to compute an initial condition `du[0]` (as mentioned above). This have to be done for the left nodes corresponding to collocation nodes. But this is not done yet. So the warning depends on `left_is_node`. Is that not correct?

Legendre seems to work, I'll do an adaption for this.

Reply:
So,

```
if left_is_node:
    raise ParameterError(msg)
```

would make more sense in this case?

Reply:
Well, for "not `left_is_node`", but yes